src/components/SequenceInput/SequenceInput.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import TimecodeInput from '../TimecodeTracker/TimecodeInput'
+import PropTypes from 'prop-types'
+
+SequenceInput.propTypes = {
+  hasDescription: PropTypes.bool,
+  hasTimeCode: PropTypes.bool,
+  inputValue: PropTypes.string,
+  isEmpty: PropTypes.bool,
+  onChange: PropTypes.func,
+  onDeleteClick: PropTypes.func,
+  onSaveClick: PropTypes.func,
+  setHasDescription: PropTypes.func,
+}
+
+export default function SequenceInput({
+  hasDescription,
+  hasTimeCode,
+  inputValue,
+  isEmpty,
+  onChange,
+  onDeleteClick,
+  onSaveClick,
+  setHasDescription,
+  textAreaValue,
+}) {
+  return (
+    <Wrapper isEmpty={isEmpty}>
+      <form onSubmit={onSaveClick}>
+        <Szenenbeschreibung
+          onChange={(event) =>
+            event.target.value.length > 0 && setHasDescription(true)
+          }
+          placeholder="Neue Szene hinzufügen"
+          name="description"
+          value={textAreaValue && textAreaValue}
+        ></Szenenbeschreibung>
+        {isEmpty && !hasDescription ? (
+          <Info>Szenenbeschreibung fehlt</Info>
+        ) : (
+          <Info hasDescription={hasDescription}>_</Info>
+        )}
+        <TimecodeInput
+          title="Szenenlänge"
+          inputValue={inputValue}
+          onChange={onChange}
+        />
+        {isEmpty && !hasTimeCode ? (
+          <Info>Timecode fehlt oder Fehlerhaft</Info>
+        ) : (
+          <Info hasTimeCode={hasTimeCode}>_</Info>
+        )}
+        <Actions>
+          <Delete onClick={onDeleteClick} type="reset">
+            LÖSCHEN
+          </Delete>
+          <Save>SPEICHERN</Save>
```

Comment:
I would use toUpperCase() Method instead of writing in all caps.

Reply:
I typed the value manually in, so it would be more code to use the string method on this. If this would be changing dynamically, then using toUpperCase() would be make sense.

Reply:
You are probably right on this. Just thought writing in all caps would be bad practice.